src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import './index.css';
+// eslint-disable-next-line import/extensions
```

Comment:
- [ ] Please remove this eslint disable rule.

Reply:
done